### PR TITLE
[ADD]: new module l10n_ec_ote

### PR DESCRIPTION
**Pull request by celm1990**
* add data for city and parish for Ecuador

### CODE REVIEW

**Review by jorge-luis-dev — APPROVED** [review]
Gracias, revisado y aprobado

File: l10n_ec_ote/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "OTE for Ecuador",
+    "summary": "OTE for Ecuador",
+    "category": "Localization",
+    "author": "Odoo Community Association (OCA)",
```
Comment (simahawk) [inline l10n_ec_ote/__manifest__.py]: Shouldn´t you add yourself as author too?

Comment (celm1990) [reply l10n_ec_ote/__manifest__.py]: @simahawk I add CONTRIBUTORS and Odoo-EC as Author, is OK?

**Review by rvjaraj — APPROVED** [review]
Revisado

**Pull request merged**